packages/core/src/stateUtils.ts

```diff
@@ -243,7 +241,6 @@ export function getValue<TC, TE extends EventObject>(
   configuration: Configuration<TC, TE>
 ): StateValue {
   const config = getConfiguration([rootNode], configuration);
```

Comment:
im not sure why this accepts 2 arguments, i've removed the "prevStateNodes" one and all tests pass without it, should I remove it in a followup PR?

Reply:
@davidkpiano what is the point of `getConfiguration`? I'm not sure how why it accepts prev & current list of state nodes.